tests/contrib/hahelpers/test_cluster_utils.py

```diff
@@ -531,7 +531,7 @@ def test_distributed_wait(self, log, modulo_distribution, sleep,
         is_leader.return_value = True
         cluster_utils.distributed_wait(modulo=9, wait=23)
         modulo_distribution.assert_not_called()
-        sleep.assert_called_with(0)
+        sleep.assert_not_called(0)
```

Comment:
It would probably be good to check that the looping does actually work here.  i.e. verify that it works for an odd number of "not divisible by" 60 seconds wait.